cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -215,6 +241,69 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
         return points;
     }
 
+    private runImageModifier():void {
+        const { activeImageModifiers } = this.state;
+        if (activeImageModifiers.length !== 0) {
+            const {
+                frameData, states, curZOrder, canvasInstance, frame,
+            } = this.props;
+            canvasInstance.configure({ forceFrameUpdate: true });
+            const canvas: HTMLCanvasElement | undefined = window.document.getElementById('cvat_canvas_background') as
+                | HTMLCanvasElement
+                | undefined;
+            if (!canvas) {
+                throw new Error('Element #cvat_canvas_background was not found');
+            }
+            const { width, height } = canvas;
+            const context = canvas.getContext('2d');
+            if (!context) {
+                throw new Error('Canvas context is empty');
+            }
+            const imageData = context.getImageData(0, 0, width, height);
+            const newImageData = activeImageModifiers.reduce((oldImageData, activeImageModifier) =>
+                activeImageModifier.modifier.processImage(oldImageData, frame), imageData);
+            frameData.imageData = newImageData;
+            canvasInstance.setup(frameData, states, curZOrder);
+            canvasInstance.configure({ forceFrameUpdate: false });
+        }
+    }
+
+    private imageModifier(alias: string):ImageProcessing|undefined {
+        const { activeImageModifiers } = this.state;
+        for (const elem of activeImageModifiers) {
+            if (elem.alias === alias) {
+                return elem.modifier;
+            }
+        }
+        return undefined;
+    }
+
+    private disableImageModifier(alias: string):void {
+        const { activeImageModifiers } = this.state;
+        let index = -1;
+        for (let i = 0; i < activeImageModifiers.length; i++) {
+            if (activeImageModifiers[i].alias === alias) {
+                index = i;
+                break;
+            }
+        }
```

Comment:
Look at Array.prototype.find()

Reply:
For this case Array.prototype.findIndex is good choice